pytorch_run.sh

```diff
@@ -202,7 +202,7 @@ fi
 
 if [ $stage -le 7 ]; then
 
-  python extract.py $modelDir $trainFeatDir $trainXvecDir
+  #python extract.py $modelDir $trainFeatDir $trainXvecDir
```

Comment:
Any reason why training embedding extraction is skipped?

Reply:
This is another mistake that I made. 
I was too lazy to check this minor points.
I will amend the commit again...! 
Thanks for pointing this out.